KeyVaultCA.Web/appsettings.json

```diff
@@ -1,10 +1,13 @@
-{
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
-    }
-  },
-  "AllowedHosts": "*"
-}
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },

```

Comment:
Should we use the 'Information' level for capturing AppInsights logs? If yes, the following snippet has to be added, because the default level is 'Warning' as per this doc https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core#how-do-i-customize-ilogger-logs-collection

    "ApplicationInsights": {
      "LogLevel": {
        "Default": "Information"
      }
}

Reply:
"Microsoft": "Warning" applies only to Assemblies with namespaces that have this pattern: "Microsoft.*"
So I think the current defaults are ok and can be changed by any adopter anyway